specs/RuntimeSpec/Fees/Fees.md

```diff
@@ -53,6 +54,7 @@ Here is the list of actions and their corresponding fees:
     - the base fee [`function_call_cost`](/GenesisConfig/RuntimeFeeConfig/ActionCreationConfig.md#function_call_cost)
     - the fee per byte of method name string and per byte of arguments with the fee [`function_call_cost_per_byte`](/GenesisConfig/RuntimeFeeConfig/ActionCreationConfig.md#function_call_cost_per_byte).
     To compute the number of bytes for a function call action `function_call_action` use `function_call_action.method_name.as_bytes().len() + function_call_action.args.len()`
+    - **Note**: owner of corresponding contract gets 30% of the fee as a reward for a possibility to invoke their function
```

Comment:
Could you be more specific here about "the fee"? There are many different fees mentioned here.